.travis.yml

```diff
@@ -28,15 +28,17 @@ jobs:
         - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
         - PHPUNIT_TEST=1
         - PHPUNIT_SUITE="cms"
-    - php: 7.4
+    - php: 8.0
       env:
         - DB=MYSQL
         - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
         - PHPUNIT_TEST=1
         - PHPUNIT_SUITE="framework"
-    - php: 8.0
+    - php: 8.1.0
```

Comment:
Does it need to be exactly `8.1.0` or `8.1` would work the same way?

Reply:
According to a thread on travis (sorry don't have link handy) it needed to be 8.1.0 for whatever reason